plotly/dashboard_objs/dashboard_objs.py

```diff
@@ -0,0 +1,441 @@
+        'shareKey': shareKey,
+        'title': title
+    }
+    return box
+
+
+def _container(box_1=None, box_2=None, size=MASTER_HEIGHT,
+               sizeUnit='px', direction='vertical'):
+    if box_1 is None:
+        box_1 = _empty_box()
+    if box_2 is None:
+        box_2 = _empty_box()
+
+    container = {
+        'type': 'split',
+        'size': size,
+        'sizeUnit': sizeUnit,
+        'direction': direction,
+        'first': box_1,
+        'second': box_2
+    }
+    return container
+
+dashboard_html = ("""
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <style>
+      body {{
+        margin: 0px;
+        padding: 0px;
+      }}
+    </style>
+  </head>
+  <body>
+    <canvas id="myCanvas" width="400" height="400"></canvas>
+    <script>
+      var canvas = document.getElementById('myCanvas');
+      var context = canvas.getContext('2d');
+      <!-- Dashboard -->
+      context.beginPath();
+      context.rect(0, 0, {width}, {height});
+      context.lineWidth = 2;
+      context.strokeStyle = 'black';
+      context.stroke();
+      </script>
+  </body>
+</html>
+""".format(width=MASTER_WIDTH, height=MASTER_HEIGHT))
+
+
+def _draw_line_through_box(dashboard_html, top_left_x, top_left_y, box_w,
+                           box_h, direction='vertical'):
+    is_horizontal = (direction == 'horizontal')
+
+    if is_horizontal:
+        new_top_left_x = top_left_x + box_w / 2
+        new_top_left_y = top_left_y
+        new_box_w = 1
+        new_box_h = box_h
+    else:
+        new_top_left_x = top_left_x
+        new_top_left_y = top_left_y + box_h / 2
+        new_box_w = box_w
+        new_box_h = 1
+
+    html_box = """<!-- Draw some lines in -->
+          context.beginPath();
+          context.rect({top_left_x}, {top_left_y}, {box_w}, {box_h});
+          context.lineWidth = 1;
+          context.strokeStyle = 'black';
+          context.stroke();
+    """.format(top_left_x=new_top_left_x, top_left_y=new_top_left_y,
+               box_w=new_box_w, box_h=new_box_h)
+
+    index_for_new_box = dashboard_html.find('</script>') - 1
+    dashboard_html = (dashboard_html[:index_for_new_box] + html_box +
+                      dashboard_html[index_for_new_box:])
+    return dashboard_html
+
+
+def _add_html_text(dashboard_html, text, top_left_x, top_left_y, box_w, box_h):
+    html_text = """<!-- Insert box numbers -->
+          context.font = '{font_size}pt Times New Roman';
+          context.textAlign = 'center';
+          context.fillText({text}, {top_left_x} + 0.5*{box_w}, {top_left_y} + 0.5*{box_h});
+    """.format(text=text, top_left_x=top_left_x, top_left_y=top_left_y,
+               box_w=box_w, box_h=box_h, font_size=FONT_SIZE)
+
+    index_to_add_text = dashboard_html.find('</script>') - 1
+    dashboard_html = (dashboard_html[:index_to_add_text] + html_text +
+                      dashboard_html[index_to_add_text:])
+    return dashboard_html
+
+
+class Dashboard(dict):
+    def __init__(self, content=None):
```

Comment:
A docstring with a simple example here would be great. That way, a user can just call help on this object and know what to do:

![image](https://cloud.githubusercontent.com/assets/1280389/23555893/965db1a6-fff8-11e6-9617-c211d355550e.png)